backend/billing/views.py

```
@@ -0,0 +1,709 @@
+    'DC': 6.00,
+}
+
+
+def get_paypal_access_token():
+    """Get PayPal OAuth access token"""
+    client_id = getattr(settings, 'PAYPAL_CLIENT_ID', None)
+    client_secret = getattr(settings, 'PAYPAL_CLIENT_SECRET', None)
+    mode = getattr(settings, 'PAYPAL_MODE', 'sandbox')
+    
+    if not client_id or not client_secret:
+        raise ValueError("PayPal credentials not configured")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)